tests/OmniSharp.Tests/IntellisenseFacts.cs

```diff
@@ -21,11 +63,11 @@ public Class1()
                     }";
 
             var completions = await FindCompletionsAsync(source);
-            ContainsCompletions(completions.Take(2), "WindowLeft", "WriteLine");
+            ContainsCompletions(completions.Select(c => c.CompletionText).Take(2), "WindowLeft", "WriteLine");
         }
 
         [Fact]
-        public async Task Returns_sub_sequence_completions()
+        public async Task ReturnsSubSequenceCompletions()
```

Comment:
Both methods are acceptable. We have both in the project. We should probably pick one and stick with it. Which one you want?